doc-Community_Collaboration_Guide/topics/structure.adoc

```diff
@@ -322,9 +322,27 @@ Placeholder text.
 [[asciibinder]]
 ==== AsciiBinder
 
-Placeholder text.
+In addition to providing a system to structure and maintain documentation for multiple brands using a single repository, AsciiBinder also provides the ability to preview and package documentation as a set of HTML files that you can publish on a website.
 
-Reference::https://placeholder.url[]
+Run the following command to build a preview of the documentation:
```

Comment:
Do we want to mention how to install AscciBinder or where to find the software if you don't have it installed?

Reply:
@nengard - Good question!

The 'Supported Tools' section goes into a bit more detail about AsciiBinder, and provides a link to the website itself. Perhaps that would be a good place to introduce this.

What do you think?